src/Facebook/GraphNodes/GraphAlbum.php

```diff
@@ -110,9 +100,9 @@ public function getDescription()
     }
 
     /**
-     * Returns profile that created the album.
+     * Returns the profile that created the album.
      *
-     * @return GraphUser|null
+     * @return GraphNode|null
```

Comment:
from the Graph API metadata
```
{
        "name": "from",
        "description": "The profile that created the album",
        "type": "user|page"
      },
```

The album may created by User or Page, so it must be returned as `GraphNode`, or as I suggest before, we can have something called `GraphProfile` as a super class of `GraphUser` and `GraphPage` to some common functionality like name, username, picture, ... etc.

So I remove the auto cast, and change the return type to `GraphNode` until we have another option?